docs/apache-airflow/core-concepts/index.rst

```diff
@@ -18,7 +18,7 @@
 Core Concepts
 =============================
 
-Here you can find detailed documentation about each one of Airflow's core concepts and how to use them, as well as a high-level :doc:`architectural overview <overview>`.
+Here you can find detailed documentation about each one of the core concepts of Apache Airflow™ and how to use them, as well as a high-level :doc:`architectural overview <overview>`.
```

Comment:
Are we registering the trademarks for `Apache Airflow` or just `Airflow`?

Reply:
Just aiirflow. But you should read it as Apache (Airflow™)

Reply:
Generally speaking it's a bit of dualism:
1) ASF expects us to use Apache Airflow in prominent places where it is mentioned for the first time on page /paragraph or chapter.
2) The trademark people will expect to see installation page and there specifically should be "Airlfow™" without Apache.

The 1)  is "Brand" , the 2) is "Trademark" and those two are slightly different and serve slightly different purpose. 

We will never get it *right* or *clearly defined* IMHO - the Trademark/Brand does not really `compile` in a consistent way I am afraid - my attempt is to make "best effort" to pass the Trdemark registration and keep ASF branding guidelines fulfilled.

Not water + fire, but close.

Reply:
okay, understood, thank you!